trackingC_setup.Rmd

```diff
@@ -235,7 +244,7 @@ run_hector <- function(pdata, c) {
   # Access and save tracking data, model outputs
   out <- list()
   out$tdata <- get_tracking_data(c) %>% filter(year %in% OUTPUT_YEARS)
-  out$results <- fetchvars(c, OUTPUT_YEARS)
+  out$results <- fetchvars(c, OUTPUT_YEARS, c("Ca", "Ftot", "FCO2", "Tgav", "luc_emissions"))
```

Comment:
No need to retrieve LUC emissions, is there?

Reply:
I pulled them out to check for myself if the values being returned were reasonable, but if we aren't planning on doing any visualizations I can go back to just the defaults.